docs/tutorial/imagestack.rst

```diff
@@ -5,113 +5,143 @@ Image Stacks
-:meth:`~lumicks.pylake.ImageStack.crop_by_pixels()` to follow the numpy
-convention (rows and then columns).
-
-Cropping can be useful, for instance, after applying color alignment to RGB images as the edges
-can become corrupted due to interpolation artifacts.
+The first argument is the color channel that you wish to plot. All additional arguments must be supplied as keyword arguments.
 
 You can also plot only a single color channel. Note that here we pass some additional formatting
 arguments, which are forwarded to :func:`plt.imshow() <matplotlib.pyplot.imshow()>`::
 
-    stack_roi.plot(channel="red", cmap="magma", adjustment=lk.ColorAdjustment(550, 800))
+    plt.figure()
+    stack_roi.plot(channel="red", cmap="magma")
+    plt.show()
 
-.. image:: correlatedstack_red.png
+.. image:: figures/imagestack/imagestack_red.png
 
-Full color RGB images are automatically reconstructed using the alignment matrices
-from Bluelake if available. This functionality can be turned off with the optional
-`align` keyword. Note that the align parameter has to be provided as a keyworded argument (i.e. `align=False`)::
+If the `channel` argument is not provided, the default behavior is `"rgb"` for 3-color images. For single-color
+images, this argument is ignored as there is only one channel available.
 
-    stack2 = lk.ImageStack("wf.tiff", align=False)
-    stack2.plot()
+Sometimes a few bright pixels can dominate the image. When this is the case, it may be beneficial to manually set the color limits
+for each of the channels. This can be accomplished by providing a :class:`~lumicks.pylake.ColorAdjustment` to plotting or export functions::
 
-.. image:: correlatedstack_raw.png
+    plt.figure()
+    stack_roi.plot("rgb", adjustment=lk.ColorAdjustment([100, 100, 100], [185, 200, 200]))
+    plt.show()
 
-You can obtain the image stack data as a :class:`numpy <numpy.ndarray>` array using the
-:meth:`~lumicks.pylake.ImageStack.get_image()` method::
+.. image:: figures/imagestack/imagestack_adjust_absolute.png
 
-    red_data = stack.get_image(channel="red") # shape = [n_frames, y_pixels, x_pixels]
-    rgb_data = stack.get_image(channel="rgb") # shape = [n_frames, y_pixels, x_pixels, 3 channels]
+By default the limits should be provided in absolute values, although percentiles can be used instead for convenience::
 
-If the `channel` argument is not provided, the default behavior is `"rgb"` for 3-color images. For single-color
-images, this argument is ignored as there is only one channel available.
+    plt.figure()
+    stack_roi.plot("rgb", adjustment=lk.ColorAdjustment(20, 99, mode="percentile"))
+    plt.show()
+
+.. image:: figures/imagestack/imagestack_adjust_percentile.png
 
+There are also a number of custom colormaps for plotting single channel images. These are available from :data:`~lumicks.pylake.colormaps`;
+the available colormaps are: `.red`, `.green`, `.blue`, `.magenta`, `.yellow`, and `.cyan`.
```

Comment:
Might be worth moving this up to where you use the `magma` colormap. I imagine it would flow a tiny bit better.